README.md

```diff
@@ -1,28 +1,69 @@
 # Nitric CLI
+
 ![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/nitrictech/cli/Tests/develop?label=tests)
 ![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/nitrictech/cli)
 ![GitHub all releases](https://img.shields.io/github/downloads/nitrictech/cli/total)
 
-
 ## Overview
 
-Nitric provides a command-line interface (CLI) to assist with various tasks when building applications with [Nitric](https://nitric.io). To view a list of available commands, you can call the CLI without specifying any subcommands or arguments:
+Nitric provides a command-line interface (CLI) to assist with various tasks when building applications with the Nitric framework. For more information, check out the main [Nitric repo](https://github.com/nitrictech/nitric).
 
-```bash
-nitric
-```
+## Installation
+
+The Nitric CLI is free to [download and install](https://nitric.io/docs/installation).
+
+## Purpose
+
+The Nitric CLI performs 3 main tasks:
+
+- Create new projects
+- Run apps locally for testing and development
+- Deploy to the cloud you choose
+
+## Common Commands
+
+Common commands in the CLI that you’ll be using:
 
-### Help and Documentation
+- nitric down            : Undeploy a previously deployed stack, deleting resources
+- nitric list            : List all project stacks and their status
+- nitric run             : Run your project locally for development and testing
+- nitric stack new       : create a new nitric stack
+- nitric up              : Create or update a deployed stack
 
-Each command is self documented and provides a "help" interface describing the usage, arguments and options for the command. Use the `help` command to view the help information for any other command:
+## Help with Commands
+
+Each command is self documented and provides a "help" interface describing the usage, arguments and options for the command. Use the help command to view the help information for any other command:
+
+Example displaying help for the `new` command
 
 ```bash
-# Example displaying help for the `stack up` command
-nitric help stack up
+nitric new --help
 ```
 
-### Bash Completion
+## Complete Reference
 
-```bash
-eval "$(nitric completion bash)"
-```
\ No newline at end of file
+Documentation for all available commands:
+
+- nitric new [projectName] [templateName] [handlerGlob] : Create a new project
+- nitric run             : Run your project locally for development and testing
+- nitric stack           : Manage stacks (project deployments)
```

Comment:
Can we change the definition of stack to be 'a collection of resources to be deployed' or something a little more descritive?